common/changes/@itwin/core-backend/jiarui-cloneMapTest_2024-06-13-16-02.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "add a special case where Pattern is null in clone maps test",
```

```suggestion
      "comment": "",
```

Change logs are for users. Users don't care about internal stuff like tests.